static/script-tests/tests/devices/broadcastsource/basetvsourcetest.js

```diff
@@ -140,12 +140,12 @@
         });
     };
 
-    this.baseTvSource.prototype.testBaseBroadcastSourceIsBroadcastSourceSupportedReturnsTrue = function(queue) {
+    this.baseTvSource.prototype.testBaseBroadcastSourceDoesNotModifyIsBroadcastSourceSupportedOnDevice = function(queue) {
```

Comment:
I think we should delete this test rather than modify it. It does not now test what it says; it says it 'makes sure the device prototype isn't modified', but it doesn't actually do that. Instead it 'makes sure that the isBroadCastSupported function returns false'. Thats not the same thing. I can see the value in driving out the change by modifying this test, but I think it should be deleted now :-)